client/components/Home.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { render } from 'react-dom';
+
+class Home extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+
+  //functionality for input field and button
+
+    // send this data to our API and potentially get info back  
+    // send a request to /data endpoint with a JSON object 
+    // { address: address}
+    // make a fetch request to the endpoint "/election"
+    // send the  JSON object
+    // use .thens and harness the data that is returned
+    // use the data in some way
+
+  //render method will render header, form, and button and functionality will be added as attirutes in renderings
+  render() {
+    return (
+      <div>
+        <header className="header"><h1>Who's On The Ballot?</h1></header>
+        <form className="inputForm">
+            <input
+              id="addressInput"
+              placeholder="Enter your address here"
+              type='text'
+          />
+          <button onClick={(event) => this.props.addressSearch(event)}>
```

Comment:
since props is thrown into the constructor for the Home class component, it can be accessed solely through props.addressSearch. If props wasn't passed into the constructor, that's when you would need to refer to it with this.props.

Reply:
this led to an interesting discussion.  we changed it to props.addressSearch